src/modules/methods.js

```diff
@@ -0,0 +1,40 @@
+export default class Methods {
+  constructor(localStorageItem, items) {
+    this.localStorageItem = localStorageItem;
+    this.items = items || [];
+  }
+
+    addItem = (description) => {
+      const completed = false;
+      const index = this.items.length + 1;
+      this.items.push({
+        index,
+        description,
+        completed,
+      });
+      this.items.sort((a, b) => a.index - b.index);
+      localStorage.setItem(this.localStorageItem, JSON.stringify(this.items));
+    };
+
+    removeItem = (index) => {
+      const deleteIndex = index - 1;
+      this.items.splice(deleteIndex, 1);
+      const newItems = this.items.map((item, index) => ({ ...item, index: index + 1 }));
+      this.items = newItems;
+      this.items.sort((a, b) => a.index - b.index);
+      localStorage.setItem(this.localStorageItem, JSON.stringify(this.items));
+    };
+
+    updateItem = (index, property, value) => {
+      const newItems = this.items.map((item) => {
+        if (item.index === parseInt(index, 10)) {
+          item[property] = value;
+        }
+        return item;
+      });
+      this.items = newItems;
+      localStorage.setItem(this.localStorageItem, JSON.stringify(this.items));
+    };
```

Comment:
- [ ] Everything looks great and is working nicely, great job 💯 However, the edited description is not being saved in local storage when I tested as you can see below, kindly fix this issue 👍 

![Animation](https://user-images.githubusercontent.com/50721479/207706797-d0762617-c4e7-4031-aca5-90523c6ef12d.gif)